lib/cart.rb

```diff
@@ -0,0 +1,42 @@
+require_relative "store"
+require_relative "billing"
+
+
+class Cart
+  attr_reader :item_names, :order, :calculator
+
+  def initialize(calculator)
+    @order = Hash.new(0)
+    @item_names = Store::ITEMNAMES
+    @calculator = calculator
+  end
+
+  def enter_order
```

Comment:
The methods which need not be exposed needs to be kept in private.
Encapsulation is one of the major core concepts of OOPS